packages/babel-helpers/src/helpers/asyncIterator.ts

```diff
@@ -96,7 +96,7 @@ function AsyncFromSyncIterator<T, TReturn = any, TNext = undefined>(s: any) {
     },
     throw: function (maybeError?: any) {
       var thr = this.s.return;
-      if (thr === undefined) return Promise.reject(maybeError);
+      if (thr === undefined) return Promise.reject(maybeError as Error);
```

Comment:
Given that `maybeError` here could be also something that is _not_ an error, I would prefer to use an `eslint-disable-next-line` comment rather than a "wrong" cast.